website/docs/apps/autoplot_app/README.md

```diff
@@ -186,18 +231,34 @@ to avoid interruptions in longer operations.
 
 If you encounter issues with autonomy or control:
 
+![icon bar](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/d69c2611-a3ad-4bb5-9db2-23a77c6a73ec)
+
 - Verify all status indicators in the Autoplot app's
 top bar are active and showing the expected states.
-- Check the [**Filter App**](/docs/apps/filter_app)
-to confirm that calibration and GPS settings are up-to-date.
+- Confirm that [**IMU Calibration**](/docs/apps/launcher/#imu-calibration)
+   and [**GPS NTRIP**](/docs/apps/launcher/#gps-ntrip) settings
+   are set correctly and up-to-date.
 - If the robot does not respond to WAD keyboard commands,
 ensure that no other tasks are running and
-that the robot is in automode.
+the robot's Dashboard is set to [**Auto Control**](/docs/dashboard/dashboard-user-guide#auto-control).
 
 ![filter error](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/9e6ab78d-dd52-4d86-9688-53b7ee9bff66)
 
+If the filter is not converged, the icon on the
+ navigation bar will show up with a line cutting across it.
+  Clicking the icon will help you resolve the error's details.
+
 ![gps error](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/c026d6fb-c86e-43d9-b8ca-928bd2ed5515)
 
+If the robot is not receiving GPS messages the icon
+ on the navigation bar will show up with a line cutting across it.
+   Clicking the icon will help you resolve the error's details.
+
 ![fail](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/9bd8b212-3286-4bac-9e40-2ec86b9ff11d)
 
+Occasionally, track following might fail. An error
+ message will appear and the **Start Path** button
+  will switch to **Error:Check status**. In this case make sure that you
+  have properly carried out an [**IMU Calibration**](/docs/apps/launcher/#imu-calibration).
```

Comment:
I'm not sure the IMU Calibration is the only way to fix errors. I think we can be less specific here as we don't have great documentation on track failure statuses just yet.